test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,82 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const chai = require('chai');
+chai.use(require('chai-subset'));
+const { StatusCodes } = require('http-status-codes');
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'stefymojica';
+const repository = 'workshop-api-testing-js';
+
+describe('Github Api Test', () => {
+  describe('Authentication', () => {
+    it('Via OAuth2 Tokens by Header', async () => {
+      const response = await axios.get(`${urlBase}/repos/${githubUserName}/${repository}`, {
+        headers: {
+          Authorization: `${process.env.ACCESS_TOKEN}`
+        }
+      });
+
+      expect(response.status).to.equal(StatusCodes.OK);
+      expect(response.data.description).equal('This is a Workshop about API Testing in JavaScript');
+    });
+  });
+});
+
+describe('Api Github', () => {
+  it('Consume GET service', async () => {
+    const response = await axios.get('https://api.github.com/users/aperdomob', {
+      headers: {
+        Authorization: `${process.env.ACCESS_TOKEN}`
+      }
+    });
+
+    expect(response.data.name).to.equal('Alejandro Perdomo');
+    expect(response.data.company).to.equal('Perficient Latam');
+    expect(response.data.location).to.equal('Colombia');
+  });
+
+  it('jasmine-json-report repository', async () => {
+    const response = await axios.get('https://api.github.com/users/aperdomob/repos', {
+      headers: {
+        Authorization: `${process.env.ACCESS_TOKEN}`
+      }
+    });
+    const found = response.data.find((element) => element.name === 'jasmine-json-report');
+
+    expect(found.name).to.equal('jasmine-json-report');
+    expect(found.visibility).to.equal('public');
+    expect(found.description).to.equal('A Simple Jasmine JSON Report');
+  });
+
+  it('Download repository', async () => {
+    const response = await axios.get('https://api.github.com/repos/aperdomob/jasmine-json-report', {
+      headers: {
+        Authorization: `${process.env.ACCESS_TOKEN}`
+      }
+    });
+    const url = `${response.data.svn_url}/archive/refs/heads/master.zip`;
+    const ResponseUrl = await axios.get(url);
+    const zip = ResponseUrl.data;
+
+    // eslint-disable-next-line no-unused-expressions
```

Comment:
Don't disable eslint

Reply:
I have to disable the rule to avoid the error "Expected an assignment or function call and instead saw an expression." because the to.exist is not a function and can not be used as to.exist()

Reply:
You can read [chai documentation](https://www.chaijs.com/api/bdd/#method_above) and see any other possible solution.
For instance you could do `expect(zip).to.have.lengthOf.above(1);` because we know zip should be an string and should have some content (greater than 1) and will not needed to disable eslint

Reply:
You can read [chai documentation](https://www.chaijs.com/api/bdd/#method_above) and see any other possible solution.
For instance you could do `expect(zip).to.have.lengthOf.above(1);` because we know zip should be an string and should have some content (greater than 1) and will not needed to disable eslint
@veronicatofino @stefymojica

Reply:
Great, thanks.